### PR TITLE
DOCS-2014: Restores RKE2 support to CC

### PR DESCRIPTION
**Pull request by ctauchen**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-cloud_versioned_docs/version-18/release-notes/index.mdx [hunk 1]
```diff
@@ -9,8 +9,18 @@ title: Release notes
...
 
+#### Support for RKE2
+
+This release comes with support for installing Calico Cloud on RKE2 clusters.
+
+For more information, see the [requirements for RKE2](../get-started/connect/requirements/rke2.mdx)
+
 ### Known issues
 
+* For clusters running on RKE2, having the Image Assurance runtime view enabled may cause Calico Cloud upgrades to fail.
```
Comment (Glen-Tigera) [inline calico-cloud_versioned_docs/version-18/release-notes/index.mdx]: Would reword to say `will cause a connect to Calico Cloud to fail if Image Assurance Runtime View was enabled prior.`

File: calico-cloud_versioned_docs/version-18/release-notes/index.mdx [hunk 2]
```diff
@@ -9,8 +9,18 @@ title: Release notes
...
+  As a workaround, temporarily disable runtime view, complete the upgrade, and then re-enable runtime view.
```
Comment (Glen-Tigera) [inline calico-cloud_versioned_docs/version-18/release-notes/index.mdx]: Would reword to say `As a workaround, disable runtime view to complete the upgrade/connection to Calico Cloud.`

File: calico-cloud_versioned_docs/version-18/release-notes/index.mdx (hunk 1 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-18/release-notes/index.mdx]: Reworded.

File: calico-cloud_versioned_docs/version-18/release-notes/index.mdx (hunk 2 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-18/release-notes/index.mdx]: Reworded.
